lib/core/test/integration/Test/Integration/Scenario/CLI/Wallets.hs

```diff
@@ -142,7 +164,52 @@ spec = do
         err `shouldBe` "Ok.\n"
         out `shouldNotContain` "CLI Wallet"
         c `shouldBe` ExitSuccess
-
   where
     emptyWallet' :: Context t -> IO String
     emptyWallet' = fmap (T.unpack . view walletId) . emptyWallet
+
+specPorts :: SpecWith (Context t)
+specPorts = do
+    it "PORTS_01 - Can create wallet with specifying port" $ \_ -> do
+        let walletName = "Wallet created via CLI"
+        checkSimpleCreateWallet
+            (Just "9080")
+            "15"
+            walletName
+            (`shouldBe` ExitSuccess)
+            (`shouldContain` "Ok.\n")
+            (`shouldContain` walletName)
+    xit "PORTS_02 - Can create wallet with relying on default port" $ \_ -> do
+        let walletName = "Wallet created via CLI"
+        checkSimpleCreateWallet
+            Nothing
+            "15"
+            walletName
+            (`shouldBe` ExitSuccess)
+            (`shouldContain` "Ok.\n")
+            (`shouldContain` walletName)
+    it "PORTS_03 - Cannot create wallet with wrong port" $ \_ -> do
+        let walletName = "Wallet created via CLI"
+        checkSimpleCreateWallet
+            (Just "9081")
+            "15"
+            walletName
+            (`shouldBe` ExitSuccess)
+            (`shouldContain` "does not exist (Connection refused))\n")
+            (`shouldNotContain` walletName)
+
+checkSimpleCreateWallet
+    :: Maybe String
+    -> String
+    -> String
+    -> (ExitCode -> IO ())
+    -> (String -> IO ())
+    -> (String -> IO ())
+    -> IO ()
+checkSimpleCreateWallet portM mnemonicSize walletName exitCodeCheck errCheck outCheck = do
+    Stdout mnemonics <- generateMnemonicsViaCLI ["--size", mnemonicSize]
+    let passphrase = "Secure passphrase"
+    (c, out, err) <- createWalletViaCLI [walletName] mnemonics "\n" passphrase portM
```

Comment:
https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters